Controller/CRUDController.php

```diff
@@ -53,7 +53,7 @@ class CRUDController extends Controller
      *
      * @return Response with json encoded data
      */
-    protected function renderJson($data, $status = 200, $headers = array(), Request $request = null)
+    protected function renderJson($data, $status = 200, $headers = array(), Request $request)
```

Comment:
@greg0ire This method and some other of `CRUDController` should be changed to `final` on another PR IMO.

What do you think?